src/utils/formValidation.ts

```diff
@@ -1,14 +1,53 @@
-export const mustBePresent = (value: any): string | undefined =>
-  value ? undefined : 'validation.message.required';
+export const stringHasNoExtraWhitespace = (value: string): boolean =>
+  value.charAt(0) !== ' ' && value.charAt(value.length - 1) !== ' ';
```

Comment:
[trimEnd](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trimEnd) and [trimStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trimStart) might come in handy here.

Reply:
Or just [trim](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim) seems to be [well supported](https://caniuse.com/?search=trim)

Reply:
I can change this to `value.trim().localeCompare(value) === 0` if that's more readable in your opinion. It's a tiny bit slower, but that doesn't really matter

Reply:
Oh, just noticed that this function is just for checking the whitespace, in this case, both solutions are fine with me.
Initially, I was thinking something like this:
  ```tsx
  export const mustBePresent = (value: any): string | undefined =>
    value?.trim() ? undefined : 'validation.message.required';
```
Will approve it as is :)